tests/xtimer_now64_continuity/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       xtimer_now64 continuity test application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdint.h>
+
+#include "xtimer.h"
+
+#define ITERATIONS 100000
+#define MAXDIFF 1000
+
+int main(void)
+{
+    unsigned n = ITERATIONS;
```

Comment:
`ITERATIONS` is greater than `2 << 16` (which is relevant since `unsigned` is `uint16_t` on MSP-430 and AVR ;-))